classes/collector/manager.php

```diff
@@ -40,14 +40,25 @@ public static function get_collector_classes(): array {
     /**
      * Sends a metric to all enabled collectors to be recorded.
      *
-     * @param metric_item $metric_item
+     * @param metric_item $item
      */
-    public static function send_metric(metric_item $metricitem) {
+    public static function send_metric(metric_item $item) {
```

Comment:
Do you need to do this often enough for this to exist? this feels like a bit of an antipattern

Reply:
It was a pre-existing function, but I suppose at this point it can be removed.

Reply:
Done